packages/core/lib/generators/integrations/svg_integration.dart

```diff
@@ -15,18 +15,21 @@ class SvgIntegration extends Integration {
         'package:flutter/widgets.dart',
         'package:flutter_svg/flutter_svg.dart',
         'package:flutter/services.dart',
+        'package:vector_graphics/vector_graphics.dart',
       ];
 
   @override
   String get classOutput => _classDefinition;
 
   String get _classDefinition => '''class SvgGenImage {
-  const SvgGenImage(this._assetName, {this.size = null});
+  const SvgGenImage(this._assetName, {this.size = null, this.isVecFormat = false});
+  const SvgGenImage.vec(this._assetName, {this.size = null, this.isVecFormat = true});
```

Comment:
Add trailing commas in constructors.

```suggestion
  const SvgGenImage(
    this._assetName, {
    this.size = null,
    this.isVecFormat = false,
  });

  const SvgGenImage.vec(
    this._assetName, {
    this.size = null,
    this.isVecFormat = true,
  });
```